pkgs/applications/science/geometry/netgen-mesher/default.nix

```diff
@@ -0,0 +1,33 @@
+{stdenv, fetchgit, cmake, python3, opencascade, tcl, tk, libjpeg, zlib, mesa, libGLU, xorg, ffmpeg, pybind11 }: 
+
+stdenv.mkDerivation rec {
+  pname = "netgen-mesher";
+  
+  version="6.2.1909";
+
+  src = fetchgit {
```

Comment:
`fetchFromGitHub` is preferred for github repos. Many examples throughout nixpkgs to refer to.

Reply:
I tried, but turns out it uses git submodules, so fetchFromGitHub won't work here.

Reply:
Oh... I was under the impression `fetchgit` would only fetch submodules if `fetchSubmodules` was specifically set - but looks like it's default `true`.

Would be good to either explicitly set this to `true` or leave a comment with the reason for this.

Reply:
Ok! I'll add a comment. It has it's own version of pybind11 library in `external_dependencies/pybind11`. I tried to convince it to work with pybind11 from  nixpkgs, but failed. I couldn't force cmake to find it. Maybe I should try harder :smile: